test/main.js

```diff
@@ -18,6 +34,30 @@ before('initialization', function () {
 	// Turn IPC methods into no-op
 	BattleEngine.Battle.prototype.send = function () {};
 	BattleEngine.Battle.prototype.receive = function () {};
+
+	callback();
+}
+
+before('initialization', function (done) {
+	this.timeout(0); // Remove timeout limitation
+
+	// Load and override configuration before starting the server
+	var Config = require('./../config/config.js');
```

Comment:
The Travis error is because of this line. Travis runs PS for the first time, before `config.js` exists.